index.html

```diff
@@ -40,6 +40,19 @@ <h1>Alerts</h1>
         Warning! Indicates a warning that might need attention.
     </div>
     <hr>
+    <h1>Avatar</h1>
+    <!-- Avatar Circle -->
+        <img src="/Assets/developer.png"  class="avatar avatar-sm" alt="" srcset="">
```

Comment:
Here also, the `alt` attribute is not provided. You can include if time persists.